File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    var answer = 0;
    sum_odd = 0;
    sum_even = 0;
    for (let i = 0 ; i < num_list.length ; i += 2) {
        sum_odd += num_list[i];
    }
    for (let j = 1; j < num_list.length ; j += 2) {
        sum_even += num_list[j];
    }
    if (sum_odd >= sum_even) {
        answer = sum_odd;
    } else {
        answer = sum_even;
    }
    return answer;
}

File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    sum_odd = sum(num_list[i] for i in range(0, len(num_list), 2))
    sum_even = sum(num_list[i] for i in range(1, len(num_list), 2))
    return sum_odd if  sum_odd >= sum_even else sum_even